roles/sft-server/meta/main.yml

```diff
@@ -16,4 +16,8 @@ galaxy_info:
   galaxy_tags:
     - SFT
 
-dependencies: []
+dependencies:
+  - role: cloudalchemy.node-exporter
+    when: sft_node_exporter_enabled
```

Comment:
Why make this optional through a variable? Especially since as far as I can see the rest of the `sft-server` role kind of assumes thi needs to be always enabled anyway? E.g. the nginx config will just not work when this role isn't enabed.

Reply:
I decided to make it not optional and forgot to remove this. Good catch :+1: